api/client/stack/deploy.go

```diff
@@ -181,24 +181,42 @@ func deployServices(
 			cspec.User = *service.User
 		}
 
+		// TODO(nishanttotla): retrieve sendAuth from a flag
+		sendAuth := true
```

Comment:
I suggest adding an argument to `deployServices` with the value of the flag. `namespace` is also passed through from a flag:

`return deployServices(ctx, dockerCli, bundle.Services, opts.namespace)`